bgpd/bgp_routemap.c

```diff
@@ -2731,10 +2731,8 @@ route_set_ipv6_nexthop_prefer_global(void *rule, const struct prefix *prefix,
 		path = object;
 		peer = path->peer;
 
-		if ((CHECK_FLAG(peer->rmap_type, PEER_RMAP_TYPE_IN)
-		     || CHECK_FLAG(peer->rmap_type, PEER_RMAP_TYPE_IMPORT))
-		    && peer->su_remote
-		    && sockunion_family(peer->su_remote) == AF_INET6) {
+		if (CHECK_FLAG(peer->rmap_type, PEER_RMAP_TYPE_IN)
```

Comment:
Just a minor nit. Instead of removing this condition completely, It would be better to add an OR condition for address family AF_INET.

Reply:
Thanks for the comment. But if the NLRI advertised by a BGP peer is irrelevant to its address family type, why should we perform the address family check here?